docs/clear-containers-3.0-quick-install.md

```diff
@@ -0,0 +1,158 @@
+# Clear Containers 3.0 - Quick-Install
+
+## Introduction
+
+### Requirements
+
+  * [go 1.8.3](https://golang.org/)
+  * [glibc-static](https://www.gnu.org/software/libc/libc.html)
+
+### Clear Containers 3.0 components
+
+  * [Runtime](https://github.com/clearcontainers/runtime)
+  * [Proxy](https://github.com/clearcontainers/proxy)
+  * [Shim](https://github.com/clearcontainers/shim)
+  * [Virtcontainers](https://github.com/containers/virtcontainers) (**pause** binary only)
+
+**IMPORTANT:** Do not combine [Clear Containers 2.1](https://github.com/01org/cc-oci-runtime) and [Clear Containers 3.0](https://github.com/clearcontainers) Both projects ship ``cc-proxy`` and they are not compatible with each other.
+
+## Set Environment
+
+### Define GOPATH
+
+```bash
+$ export GOPATH=$HOME/go
+```
+
+### Create GOPATH Directory
+
+```bash
+$ mkdir -p $GOPATH
+```
+
+### Get the code
+
+```bash
+$ go get github.com/clearcontainers/runtime
+$ go get github.com/clearcontainers/proxy
+$ git clone https://github.com/clearcontainers/shim $GOPATH/src/github.com/clearcontainers
+```
+
+## Build and Install Components
```

Comment:
@jodh-intel  would you have a text listing the different components, their function and how they interact with one another?